docs/content/direct/packaging.md

```diff
@@ -65,19 +63,12 @@ By our development practices and doing doing any manual hacks, we maintain the a
 
 ## OCM Transport Plugin
 
-This thing is in the midst of being moved from a separate repository into the kubestellar repository. The separate repository is [github.com/kubestellar/ocm-transport-plugin](https://github.com/kubestellar/ocm-transport-plugin).
+This repository ([github.com/kubestellar/ocm-transport-plugin](https://github.com/kubestellar/ocm-transport-plugin)) is retired. Its contents have been merged into the kubestellar repository.
 
-The primary product is the OCM Transport Controller, built from generic transport controller code plus code specific to using OCM for transport.
+The primary product was the OCM Transport Controller, built from generic transport controller code plus code specific to using OCM for transport. The published artifacts, which still linger because the GitHub is all about not forgetting things, are as follows. DO NOT USE THEM.
```

Comment:
Maybe I am overthinking: Is it better to be explicit that the OCM Transport Controller is still alive? The past tense sounds to me like the OCM Transport Controller is also retired along with ks/otp repo.

Not sure whether I explained clearly. Let me also try another way: 'The OCM Transport Controller was previously built from...' is less ambiguous to me.

Reply:
Revised for clarity. PTAL.